README.md

```diff
@@ -1,3 +1,37 @@
 # go-library is a monorepo of modules
 
 The go-library is a monorepo of golang modules designed for reusability within CLI and daemon applications.
+
+## Conventions
+
+A package should be self contained within its own folder off the root of the repository.
+
```

Comment:
```suggestion
A package should be self contained within its own folder off the root of the repository.

If a package grows rapidly or versioning complex arises due to frequent package changes, it might make sense to move certain packages to their own Go modules as needed that are versioned and tagged separately.
```